data/data/baremetal/bootstrap/variables.tf

```diff
@@ -1,19 +1,9 @@
-variable "cluster_id" {
-  type        = string
-  description = "The identifier for the cluster."
+variable "control_plane_ips" {
+  type    = list(string)
+  default = []
 }
 
-variable "image" {
-  description = "The URL of the OS disk image"
-  type        = string
-}
-
-variable "ignition" {
-  type        = string
-  description = "The content of the bootstrap ignition file."
-}
-
-variable "bridges" {
-  type        = list(map(string))
-  description = "A list of network bridge maps, containing the interface name and optionally the MAC address"
+variable "bootstrap_ip" {
```

Comment:
Unused and never set.

Reply:
Removed this file since nothing was being used in it.